Assets/MixedRealityToolkit.SDK/Experimental/Features/UX/MixedRealityKeyboardBase.cs

```diff
@@ -0,0 +1,175 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using UnityEngine.UI;
+#if WINDOWS_UWP
+using Windows.UI.ViewManagement;
+using Microsoft.MixedReality.Toolkit.Input;
+using System.Collections;
+#endif
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.UI
```

Comment:
Ah shoot, just realized this and MixedRealityKeyboard.cs are not in the correct folder structure. Could you please place this and MixedRealityKeyboard under:

`Assets/MixedRealityToolkit.SDK/Experimental/MixedRelityKeyboard/`?

[Guidance here](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/ExperimentalFeatures.html#experimental-code-should-live-in-a-separate-folder)

Reply:
Just updated to the latest dev branch and all the changes were made under `Assets/MixedRealityToolkit.SDK/Experimental/Features/UX/...`

Reply:
We actually want to move to have all experimental features match structure `Assets/<MRTKFolder>/Experimental/<ExperimentalFeatureName>` to make the features easier to discover, following [this guidance](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/ExperimentalFeatures.html#experimental-code-should-live-in-a-separate-folder). Could you please update?